### DISCUSSION TITLE
Allow for canceling event bubbling when using `useClick`

### DISCUSSION THREAD

**Discussion by joshferrell** [floating-ui/floating-ui#3093] [Ideas]
I'd like to use the "mouseDown" event in the useClick library. However, it does not allow me to cancel event bubbling to parent events. This is important for complex apps like Figma where events could potentially bubble to the canvas, or in my case, we require components to explicitly prevent bubbling so that our draggable window component can be dragged from anywhere.

**Comment by olsonpm** [comment]
In my case I'm looking to disable bubbling for onHover - which, from my understanding, requires a click on mobile.

My use case is label icons within a link.  On desktop, hovering over these labels provides more info.  On mobile, I would like for users to touch these labels without navigating.

---

edit: fwiw in my case this was easy to accomplish by calling `evt.preventDefault` in both onClick and onMouseDown events for a parent element.
